docs/ibexa_engage/ibexa_engage.md

```diff
@@ -0,0 +1,53 @@
+---
+description: Ibexa Engage helps to engage audiences and collect valuable data by using interactive content.
+---
+
+# Ibexa Engage
```

Comment:
I'm just wondering about the name of this section, does it make sense to add prefix "Ibexa" or just Engage? Most sections don't contain it (Perso, Commerce, DAM etc.). :)

Reply:
I'd keep Ibexa Engage - even on the welcome page we use _"Welcome to **Ibexa Engage**"_, _"Discover **Ibexa Engage**"_, etc. 
Check Ibexa Cloud or Ibexa CDP - we use full name as well.